bin/build-driver.sh

```diff
@@ -19,5 +19,4 @@ check_clojure_cli
 source "./bin/clear-outdated-cpcaches.sh"
 clear_outdated_cpcaches
 
-cd bin/build-drivers
-clojure -M -m build-driver $@
+clojure -X:build:drivers:build/driver :driver $@
```

Comment:
I switched some of these things to use cool new `-X` instead of gross old `-M`.

When I wrote all this `-M` stuff I was brand new to Clojure CLI. Now I actually know what I'm doing